docs/uptime/overview.asciidoc

```diff
@@ -0,0 +1,40 @@
+[role="xpack"]
+[[uptime-overview]]
+
+== Uptime Overview page
+
+The Overview page is intended to help you quickly identify and diagnose outages and
```

Comment:
The Kibana docs try not to use "page" and other similar terms whenever possible. 
```suggestion
The Uptime overview is intended to help you quickly identify and diagnose outages and
```